jenkins-config/managed-config-files/mavenSettings/dentos-settings/content

```diff
@@ -0,0 +1,257 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
```

Comment:
You should rip out all the boiler plate comments as well as all of the sections for this file. It should end up being the <?xml header line and <settings open and </settings> close

Also, move one out of the managed-config-files directory to a directory called managed-config-templates and name it mavenSettings-content and then symlink to it for all of the other mavenSettings content files.

Reply:
got it! thanks!